src/lib/components/Modal/README.mdx

```diff
@@ -273,86 +619,48 @@ component.
-                nascetur ridiculus mus. Donec quam felis, ultricies nec,
-                pellentesque eu, pretium quis, sem.
-              </p>
-              <p>
-                Nulla consequat massa quis enim. Donec pede justo, fringilla
-                vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus
-                ut, imperdiet a, venenatis vitae, justo.
-              </p>
-              <p>
-                Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean
-                commodo ligula eget dolor. Aenean massa.
-              </p>
-              <p>
-                Cum sociis natoque penatibus et magnis dis parturient montes,
-                nascetur ridiculus mus. Donec quam felis, ultricies nec,
-                pellentesque eu, pretium quis, sem.
-              </p>
-              <p>
-                Nulla consequat massa quis enim. Donec pede justo, fringilla
-                vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus
-                ut, imperdiet a, venenatis vitae, justo.
-              </p>
-              <p>
-                Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean
-                commodo ligula eget dolor. Aenean massa.
-              </p>
-              <p>
-                Cum sociis natoque penatibus et magnis dis parturient montes,
-                nascetur ridiculus mus. Donec quam felis, ultricies nec,
-                pellentesque eu, pretium quis, sem.
-              </p>
-              <p>
-                Nulla consequat massa quis enim. Donec pede justo, fringilla
-                vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus
-                ut, imperdiet a, venenatis vitae, justo.
-              </p>
-              <p>
-                Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean
-                commodo ligula eget dolor. Aenean massa.
-              </p>
-              <p>
-                Cum sociis natoque penatibus et magnis dis parturient montes,
-                nascetur ridiculus mus. Donec quam felis, ultricies nec,
-                pellentesque eu, pretium quis, sem.
-              </p>
-              <p>
-                Nulla consequat massa quis enim. Donec pede justo, fringilla
-                vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus
-                ut, imperdiet a, venenatis vitae, justo.
-              </p>
+              <ModalHead
+                onClose={() => setModalOpen(false)}
+                title="Delete the user?"
+              />
+              <ModalBody isScrollViewCompatible={modalScrollView}>
+                {
+                  modalScrollView
+                    ? (
+                      <ScrollView
+                        customEndShadowStyle={{
+                          background: 'radial-gradient(farthest-side at center bottom, rgba(0, 0, 0, 0.16) 0%, rgba(0, 0, 0, 0.06) 40%, rgba(0, 0, 0, 0.02) 85%, rgba(0, 0, 0, 0) 100%)'
```

Comment:
So the scrolling shadow defaults seem to be the only thing we are losing.

Could we offload the definition into constants so this code is less cluttered maybe…?

<img width="840" alt="obrazek" src="https://user-images.githubusercontent.com/5614085/175784207-c845a43e-16bc-46e1-8c13-11e6e5caa5cc.png">

Reply:
Or! Can we define the shadow constants here in the `Modal` (and make them modifiable through props) and pass them down to `ScrollView` via context?

Reply:
You are right, we are losing default configuration of ScrollView which is used within Modal.

After I read your comment, I wrapped `children` in `ModalBody` into `RUIProvider` to be able to override defaults for `ScrollView`. We use `RUIProvider` as Global Props, but It can be used anywhere. But I reverted it. Because it would override preferences of end user. Custom context does not problem neither.

There are following ways how to deal with this problem:

1. Leave it as it i.
2. Save default configuration into the constant and export it to allow user to pass this configuration using spread rest. Something like `<ScrollView {...ModalBody.scrollViewDefaultProps} />`
3. To merge `ModalBody` and `ModalContent` and implement `ScrollView` internally as it was in original implementation.
4.  To create `ModalScrollableContent` which will be only wrapper for `ScrollView`. We will then be able to configure in using Global Props. 


I kinda like 4 as it would allow Global Props configuration separately for `ScrollView` and `ModalScrollableContent`.

Reply:
The problem is that internal implementation of Modal's ScrollView has different appearance than it should have. I would also discuss it on Monday

Reply:
In #393 I'm suggesting to support three scrolling modes:

- `auto` mode that turns on scrolling for `ModalBody`, provides basic scrolling functionality.
- `custom` mode that expects `ScrollView` (or anything else if necessary) with any design props.
- `none` mode leaving scrolling to `Modal` itself.

Also, I'm introducing customizable inner separators to make design more than acceptable no matter the scrolling mode.

<img width="397" alt="obrazek" src="https://user-images.githubusercontent.com/5614085/178562187-fe00fe6a-ad82-4a7e-8bd5-cf0ccd362562.png">